email_account_validity/_utils.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# Copyright 2021 The Matrix.org Foundation C.I.C.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import random
+import re
+import string
+
+
+UNAUTHENTICATED_TOKEN_REGEX = re.compile('^[a-zA-Z]{32}$')
```

Comment:
I'd be tempted to go just `if len(renewal_token) >= 32`, but ymmv

Reply:
did you deliberately decide against this?

Reply:
Yes, I think it makes it clearer when reading the code which format is a long token supposed to be.

Reply:
if you're going to strictly validate the format of long renewal tokens, why not do the same for short ones? It feels strangely asymmetric to me.

Reply:
Fair enough.